test/functional/readconcern_tests.js

```diff
@@ -56,6 +56,52 @@ describe('ReadConcern', function() {
     }
   });
 
+  it('Should set local readConcern on db level when using createCollection method', {
+    metadata: { requires: { topology: 'replicaset', mongodb: '>= 3.2' } },
+
+    test: function(done) {
+      var listener = require('../..').instrument(function(err) {
+        test.equal(null, err);
+      });
+
+      // Contains all the apm events
+      var started = [];
+      // Get a new instance
+      var configuration = this.configuration;
+      var client = configuration.newClient(
+        { w: 1 },
+        { poolSize: 1, readConcern: { level: 'local' } }
+      );
+
+      client.connect(function(err, client) {
+        expect(err).to.not.exist;
+
+        var db = client.db(configuration.db);
+        test.deepEqual({ level: 'local' }, db.s.readConcern);
+
+        // Get a collection using createCollection
+        var createdCollection = db.createCollection('readConcernCollection');
+        // Validate readConcern
+        test.deepEqual({ level: 'local' }, createdCollection.s.readConcern);
```

Comment:
I think we can make this the end of the test. We don't need to check that the `find` below has the correct `readConcern`. This will also allow us to remove any code related to the `listener` in this test.